--- Sphinx-0.6.5-py2.6.egg_FILES/sphinx/writers/html.py	2010-01-02 00:43:38.000000000 +0200
+++ Sphinx-0.6.5-py2.6.egg_FILES/sphinx/writers/html.py	2010-05-27 02:24:33.203418028 +0300
@@ -16,6 +16,7 @@
 from docutils import nodes
 from docutils.writers.html4css1 import Writer, HTMLTranslator as BaseTranslator
 
+from sphinx import addnodes
 from sphinx.locale import admonitionlabels, versionlabels
 from sphinx.util.smartypants import sphinx_smarty_pants
 
@@ -284,6 +285,14 @@ class HTMLTranslator(BaseTranslator):
     def depart_centered(self, node):
         self.body.append('</strong></p>')
 
+    # overwritten
+    def should_be_compact_paragraph(self, node):
+        """Determine if the <p> tags around paragraph can be omitted."""
+        if isinstance(node.parent, addnodes.desc_content):
+            # Never compact desc_content items.
+            return False
+        return BaseTranslator.should_be_compact_paragraph(self, node)
+
     def visit_compact_paragraph(self, node):
         pass
     def depart_compact_paragraph(self, node):
